modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -15,27 +16,32 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens.
-* link:https://docs.openshift.com/container-platform/4.14/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
+* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Prepare AWS credentials_.
+
+
+* link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
 
 .Procedure
 
-. Create an Openshift secret from your AWS token file by entering the following commands:
+. Create an Openshift secret from your AWS token file by entering the following commands.
 
 .. Create the credentials file:
 +
-[source, terminal]
+[source,terminal]
 ----
 $ cat <<EOF > ${SCRATCH}/credentials
 [default]
-role_arn = ${ROLE_ARN}
+role_arn = ${ROLE_ARN} <1>
 web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 EOF
 ----
 
++
+<1> Replace `ROLE_ARN` with the output you saved while creating the role.
+
 .. Create the OpenShift secret:
 +
-[source, terminal]
+[source,terminal]
 ----
 $ oc -n openshift-adp create secret generic cloud-credentials \
   --from-file=${SCRATCH}/credentials
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/97602793/cb7b26d9-6473-4ed9-be21-3af88ddca0a5)
It is already `b` after the `a` in my preview.

Reply:
It's shown correctly now, maybe because the following sentense was removed.

"<1> Replace `ROLE_ARN` with the output you saved while creating the role."